pkg/api/pod/util.go

```diff
@@ -385,6 +400,7 @@ func GetValidationOptionsFromPodSpecAndMeta(podSpec, oldPodSpec *api.PodSpec, po
 		AllowInvalidTopologySpreadConstraintLabelSelector: false,
 		AllowNamespacedSysctlsForHostNetAndHostIPC:        false,
 		AllowNonLocalProjectedTokenPath:                   false,
+		AllowInvalidRecursiveReadOnlyMounts:               false,
```

Comment:
I think you don't need this at all - dropDisabled already wipes it

Reply:
fixed